apis/alloydb/v1beta1/cluster_types.go

```diff
@@ -23,6 +23,68 @@ import (
 
 var AlloyDBClusterGVK = GroupVersion.WithKind("AlloyDBCluster")
 
+// +kcc:proto=google.cloud.alloydb.v1beta.Cluster.PscConfig
+type Cluster_PSCConfig struct {
```

Comment:
Hmmm, I assume the change is made because of the acronym capitalization issue? Can I get an /lgtm for https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/3624? With this change, you shouldn't need to do manual modifications.

Reply:
Right. I'm not sure if we reached a consensus on which acronym to include in that PR, so I manually moved the Go structs in this PR to unblock the fix.